sandbox/handlers.js

```diff
@@ -36,10 +36,8 @@ const write_log = (res, log_level, options = {}) => {
 
 async function status(req, res, next) {
     res.json({
-        status: "pass",
-        ping: "pong",
-        service: req.app.locals.app_name,
-        version: req.app.locals.version_info
+        commitId: "1231231231231231231231231231231231231231",
+        versionNumber: "v1.2.34-sandbox"
     });
```

Comment:
Rather than modify the docker container response, we should change the sandbox check to call out to prescriptions-for-patients in proxies/sandbox/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml as you have done for the non sandbox request

Reply:
resolved in 67f587a2b6d5936227763b3de0d28c1fa9536eb9